my_init.d/factory_reset.sh

```diff
@@ -1,34 +1,50 @@
-#!/bin/bash -e
-
-# Debugging.
-set -x
```

Comment:
Are you sure you want to delete that? I think, if something goes wrong - the image should not start.

Reply:
The [`-x` flag](https://stackoverflow.com/a/36273740) only instructs bash to echo all commands, it does not influence whether the script fails on error. The `-e` flag is still set (see line 1).